### ISSUE TITLE
npm readme gives wrong config format in Usage section

### ISSUE THREAD

**Issue by adamlui** [azeemba/eslint-plugin-json#92]
It shows

```js
{
    export default [
      {
        files: ["**/*.json"],
        ...json.configs["recommended"],
    ];
}
```

...but should be like the format in GitHub readme without missing closing curly brace

```js
export default [
  {
    files: ["**/*.json"],
    ...json.configs["recommended"]
  }
];
```

**Comment by fnogatz** [comment]
Note that this has already been resolved (see [current version of `README.md`](https://github.com/azeemba/eslint-plugin-json/blob/543b6805bc6e84a8d3ab1099bd337a196dc5786a/README.md)) as part of https://github.com/azeemba/eslint-plugin-json/pull/89, it just has not yet been pushed to npm as well. Maybe you could do so, @azeemba, so we can close this issue here?

**Comment by azeemba** [comment]
Whoops, thanks. Will do so later this evening

**Comment by adamlui** [comment]
> Note that this has already been resolved (see [current version of `README.md`](https://github.com/azeemba/eslint-plugin-json/blob/543b6805bc6e84a8d3ab1099bd337a196dc5786a/README.md))

I know that README has the correct instruction, I said that in my OP, the issue is on npm it doesn't so it's not actually resolved

**Comment by azeemba** [comment]
Thanks for the report and the update everyone!

There was [no way](https://github.com/npm/npm/issues/7016) to update the readme without a new published version. So I have published `4.0.1` with the updated readme.
